pkg/kubelet/kubelet_pods.go

```diff
@@ -1787,7 +1787,8 @@ func allocatedResourcesMatchStatus(allocatedPod *v1.Pod, podStatus *kubecontaine
 
 			// Only compare resizeable resources, and only compare resources that are explicitly configured.
 			if hasCPUReq {
-				if !cpuReq.Equal(*cs.Resources.CPURequest) {
+				if !cpuReq.Equal(*cs.Resources.CPURequest) &&
+					(cpuReq.MilliValue() > cm.MinShares || cs.Resources.CPURequest.MilliValue() > cm.MinShares) {
```

Comment:
Add a comment here to explain why we exclude those values.

Reply:
Done.